nx.json

```diff
@@ -93,6 +93,49 @@
           "targetName": "typecheck"
         }
       }
-    }
-  ]
+    },
+    "@nx/owners"
+  ],
+  "sync": {
+    "applyChanges": true,
+    "globalGenerators": ["@nx/owners:sync-codeowners-file"]
+  },
+  "owners": {
+    "format": "github",
+    "outputPath": "CODEOWNERS",
+    "patterns": [
+      {
+        "description": "Frontend Team Projects",
+        "projects": ["tag:frontend"],
+        "owners": ["@frontend"]
+      },
+      {
+        "description": "Backend Team Projects",
+        "projects": ["tag:backend"],
+        "owners": ["@backend"]
+      }
```

Comment:
Feel free to additional owners here but this should be a good starting point at least